src-built-in/components/advancedAppLauncher/src/index.jsx

```diff
@@ -62,7 +62,7 @@ class AppLauncher extends React.Component {
 	openAppMarket() {
 		FSBL.Clients.LauncherClient.showWindow(
 			{
-				componentType: "App Catalog"
+				componentType: "Advanced App Catalog"
```

Comment:
I changed this purposefully. If someone enables the advanced app launcher, we will default to showing the advanced app catalog alongside it. By default, the link to the Advanced App catalog is hidden. So my assumption is that if a developer is going to go and uncomment that code, they know what they're doing.